src/components/Select.vue

```diff
@@ -9,18 +9,29 @@
         aria-labelledby="select-label select-button"
         :aria-expanded="areOptionsVisible"
         class="selected-btn"
-        :class="{ active: areOptionsVisible }"
+        :class="{ active: areOptionsVisible, big: type === 'big' }"
         @click="toggleOptions"
         @keyup.up.down.prevent="showOptions"
         @keyup.up.prevent="selectPrevOption"
         @keyup.down.prevent="selectNextOption"
       >
         <div class="selected">
-          <img class="item-icon" v-if="value.img" :src="value.img" alt="icon" />
-          <div class="label">
-            <div class="primary" :data-test="'value' + value.primaryText">
-              {{ value.primaryText }}
+          <div class="label" :class="{ big: type === 'big' }">
```

Comment:
What you think about add the `type` as a class directly and add a [prop validation](
![image](https://user-images.githubusercontent.com/24905681/81791306-ddb21780-9506-11ea-93ee-d7946ba76540.png)
) 
```suggestion
          <div class="label" :class="[type]">
```